app.js

```diff
@@ -724,10 +724,25 @@ function publishDiscovery( device, index ){
 					}
 */
 					var discoveryTopic = `${CONFIG.discovery_prefix}${parameter.component}/ener314rt/${object_id}/config`;
-//					var dmsg = Object.assign({ uniq_id: `${unique_id}`, "~": `${CONFIG.topic_stub}`, name: `${name}`, mf: 'energenie', sw: 'mqtt-ener314rt' },
-					var dmsg = Object.assign( { device: { name: `${device_name}`, ids: [`ener314rt-${device.deviceId}`], mdl: `${device_defaults.mdl}`, mf: 'Energenie', sw: `mqtt-ener314rt ${APP_VERSION}` }, 
-											uniq_id: `${unique_id}`, "~": `${CONFIG.topic_stub}${device.productId}/${device.deviceId}/`, name: `${entity_name}`, availability_topic: `${CONFIG.topic_stub}availability/state` },
-											parameter.config,);
+					var dmsg = Object.assign({
+						device: {
+							name: `${device_name}`,
+							ids: [`ener314rt-${device.deviceId}`],
+							mdl: `${device_defaults.mdl}`,
+							mf: `Energenie`,
+							sw: `${APP_VERSION}`
```

Comment:
Can you leave sw as is - sw: `mqtt-ener314rt ${APP_VERSION}`

Reply:
> Can you leave sw as is - sw: `mqtt-ener314rt ${APP_VERSION}`

Reverted to `- sw: mqtt-ener314rt ${APP_VERSION}`